src/utils/uuid.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function generateUuid() {
```

Comment:
This does not generate Uuid as-is, but a custom format. It should be name differently to avoid confusion.

better use https://www.npmjs.com/package/uuid if we really need uuid.

Reply:
You right, the naming isn't good. It's generate an unique identifier but not an universal. I can rename this by `generateUID`

For the npm package we want to avoid external dependance right ?

Reply:
we want to avoid specific JS framework, but dependancy to JS 'vanilla' component is ok I think.